packages/vscode-extension/src/builders/eas.ts

```diff
@@ -80,7 +87,10 @@ async function downloadAppFromEas(
   const { id, binaryUrl } = build;
 
   const tmpDirectory = await mkdtemp(path.join(os.tmpdir(), "rn-ide-eas-build-"));
-  const binaryPath = path.join(tmpDirectory, id);
+  const binaryPath =
+    platform === DevicePlatform.Android
+      ? path.join(tmpDirectory, `${id}.apk`)
```

Comment:
The error message from above says it can be in apex format too?

Reply:
according to the [documentation](https://source.android.com/docs/core/ota/apex#apex-are-apk) are apk files so adding less specific extension won't break anything